web/src/components/organisms/Project/Content/ViewsMenu/hooks.ts

```diff
@@ -41,20 +41,16 @@ export default ({ modelId, currentView, setCurrentView }: Params) => {
   });
 
   const views = useMemo(() => {
-    if (!loading && data?.view) {
+    if (loading && data?.view) {
```

Comment:
Why did you remove the exclamation mark? this is causing the initial view to not be loaded correctly.

Reply:
With exclamation mark, navigating to newly created view doesn't work.

Reply:
initial view and newly created view should be working at the same time

Reply:
I think he did it for navigating to newly created view but it cause the issue so we should do other solution.

Reply:
Sorry. I was wrong. I created prevModelId state.